.github/2.0/cookbooks/Flow.md

```diff
@@ -431,6 +431,92 @@ f = (Flow()
 
 </table>
 
+#### Commonly used arguments for distribution setup in `.add`
+
+| Name | default | Description |
+| --- | --- | --- |
+| `host` | `0.0.0.0` | The host of the machine. Can be an ip address or DNS name (e.g. `0.0.0.0`, `my_encoder.jina.ai`) |
+| `port_expose` | randomly initialized | Port of JinaD on the remote machine. |
+| `port_in` | randomly initialized | Port for incoming traffic for the Executor. |
+| `port_out` | randomly initialized | Port for outgoing traffic for the Executor. This is only used in the remote-local use-case described below. |
+| `connect_to_predecessor` | `False` | Forces a Head to connect to the previous Tail. This is only used in the remote-local use-case described below. |
+| `external` | `False` | Stops `Flow` from context managing an Executor. This allows spawning of an external Executor and reusing across multiple Flows. |
+| `uses`, `uses_before` and `uses_after` prefix | No prefix | When prefixing one of the `uses` arguments with `docker` or `jinahub+docker`, the Pod does not run natively, but is spawned inside a container. |
+
+#### Commonly used patterns for `.add`
+
+##### Local native Executor via `.py` file:
+
+```python
+class MyExecutor():...
+
+f.add(uses=MyExecutor)
+f.add(uses='MyExecutor')
```

Comment:
This is just adding from a string? I see the same line duplicated except one has quotes

Reply:
Yes you can either pass the object or the name of the Executor.

Reply:
Let's make that clear in the doc (unless I missed it somewhere)

Reply:
I did in the latest change. :+1: Thanks for pointing me at it.